tot/data_processing/scaler.py

```diff
@@ -0,0 +1,66 @@
+from dataclasses import dataclass
+
+import numpy as np
+import pandas as pd
+
+
+@dataclass
+class Scaler:
```

Comment:
Can we add an example in the docstring?

Reply:
it might make sense to add the example to `transform()` and `inverse_transform()` instead. What do you think?